### ISSUE TITLE
Screamer 2 Crashes when selecting "network game" (IPX)

### ISSUE THREAD

**Issue by OpenRift412** [joncampbell123/dosbox-x#4063]
### Describe the bug

Whenever I run Screamer 2 (both software and 3dfx versions) and try starting a multiplayer game, selecting the "Network game" option just makes a weird corrupted sort of sound and locks up the game (except the CD music is still playing).

### Steps to reproduce the behaviour

1. Launch the game
2. In the main menu, select multiplayer
3. Select "network game"

### Expected behavior

It's supposed to load the network lobby screen. Tried with DOSBox Staging and it worked fine.

### What operating system(s) this bug have occurred on?

Windows 11 22H2

### What version(s) of DOSBox-X have this bug?

DOSBox-X 2022.12.26 2022-12-26 19:55

### Used configuration

```ini
This happens even with the default configuration.
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
Try the latest nightly (development) build.
![s2_ipx](https://user-images.githubusercontent.com/68574602/223959219-24d6b169-fa7f-4eb9-8690-ef34cd5e0728.png)

**Comment by joncampbell123** [comment]
I just tested the latest code in the master branch. The IPX fix that lets DOOM and Hexen work also allows this game to work too. You'll be able to use it when the next release comes out soon.

Here's a test I did with 3 instances of the game talking to each other over IPX.
![asspoofartscreamer2](https://user-images.githubusercontent.com/6245486/228141311-877ef5ad-1b1d-4816-9817-9f18210734a2.jpg)

Yes, they are able to do multiplayer racing too.
